doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown

```diff
@@ -684,9 +684,9 @@ References {#tutorial_documentation_refs}
 - [Command reference] - supported commands and their parameters
 
 <!-- invisible references list -->
-[Doxygen]: http://www.stack.nl/~dimitri/doxygen/index.html)
-[Doxygen download]: http://www.stack.nl/~dimitri/doxygen/download.html
-[Doxygen installation]: http://www.stack.nl/~dimitri/doxygen/manual/install.html
+[Doxygen]: http://doxygen.nl
+[Doxygen download]: http://doxygen.nl/download.html
+[Doxygen installation]: http://doxygen.nl/manual/install.html
 [Documenting basics]: http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html
 [Markdown support]: http://www.stack.nl/~dimitri/doxygen/manual/markdown.html
 [Formulas support]: http://www.stack.nl/~dimitri/doxygen/manual/formulas.html
```

Comment:
Could you please update these 3 links too and others:

```
$ grep -R 'stack.nl/~dimitri' ./
./doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown:[Documenting basics]: http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html
./doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown:[Markdown support]: http://www.stack.nl/~dimitri/doxygen/manual/markdown.html
./doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown:[Formulas support]: http://www.stack.nl/~dimitri/doxygen/manual/formulas.html
./doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown:[Command reference]: http://www.stack.nl/~dimitri/doxygen/manual/commands.html
./doc/tutorials/introduction/windows_install/windows_install.markdown:-   [Doxygen](http://www.stack.nl/~dimitri/doxygen/) is a documentation generator and is the tool that will actually create the
./doc/tutorials/introduction/cross_referencing/tutorial_cross_referencing.markdown:[Doxygen](https://www.stack.nl/~dimitri/doxygen/) is a tool to generate
./doc/tutorials/introduction/cross_referencing/tutorial_cross_referencing.markdown:- [Doxygen: Linking to external documentation](https://www.stack.nl/~dimitri/doxygen/manual/external.html)
```

BTW, [Documentation page with patch](https://pullrequest.opencv.org/buildbot/export/pr/15980/docs/d4/db1/tutorial_documentation.html)

Reply:
Done. I can't believe I missed the links in the same file. I'm glad one of us is paying attention. I'll try to be more thorough.